sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -809,6 +809,39 @@ public void testSelectStarFromSelectSingleColumnWithLimitDescending() throws Exc
             new Object[]{"def"}
         )
     );
+
+    List<Object[]> expected;
+    if (NullHandling.replaceWithDefault()) {
+      expected = ImmutableList.of(
+          new Object[]{"", 1L},
+          new Object[]{"def", 1L}
+      );
+    } else {
+      expected = ImmutableList.of(
+          new Object[]{"def", 1L},
+          new Object[]{"abc", 1L}
+      );
+    }
+    testQuery(
```

Comment:
I think it's best to have one `testQuery` call per test method. So, I'd create a new method for this test.

Reply:
split the extra one already there, the one i added into separate tests, also added a few more